src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/InitializeParameter/InitializeParameterHelpers.cs

```diff
@@ -194,37 +300,4 @@ private static AccessorDeclarationSyntax RemoveThrowNotImplemented(AccessorDecla
 
         return result.WithTrailingTrivia(accessorDeclaration.Body?.GetTrailingTrivia() ?? accessorDeclaration.SemicolonToken.TrailingTrivia);
     }
-
-    public static bool IsThrowNotImplementedProperty(Compilation compilation, IPropertySymbol property, CancellationToken cancellationToken)
```

Comment:
another copy of this funciton taht was moved to a shared location and unified with the other.